pkg/component/component.go

```diff
@@ -130,7 +131,7 @@ func CreateFromGit(client *occlient.Client, params occlient.CreateArgs) error {
 	labels := componentlabels.GetLabels(params.Name, params.ApplicationName, true)
 
 	// Loading spinner
-	s := log.Spinnerf("Creating component %s", params.Name)
+	s := log.SpinnerNoSpin("Creating component " + params.Name)
```

Comment:
Why are we doing no spinner here? This should stay the same so that users know what's going on.. If we do no spin we get the awful "..." output only meant for debugging.